00-Introducing-HOOMD-blue/05-Compressing-the-System.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
<p>Remove for now since we are not adding this yet.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I replaced it with a while loop and updated the description accordingly.</p>